\graphicspath{{chapters/LiquidBiopsiesImages/}}

\chapter{Liquid biopsies in oncology}

\textbf{\textit{Written by Linda Cova}}

\section{Liquid vs Tissue biopsies}

\begin{tabular}{ | m{5cm}| m{9cm} | }
 \hline
 \textbf{Tissue Biopsy} & \textbf{Liquid Biopsy} \\
 \hline
 Accurate and detailed view of one tissue only & Landscape overview, with resolution depending on tumor burden, releasing rates, metastases and tumor heterogeneity. It is possible to get an aggregated signal of different tumor cell populations \\
 \hline
 Single tumor & Possibility of getting signal from multiple tumor masses \\
 \hline
 Signal relative to a specific point in time & At a certain point in time, but multiple serial samples can be collected \\
 \hline
 Invasive and painful for the patient, not feasible for all the tissues & Minimally invasive (so it is possible to collect samples multiple times) and can be coupled with a routine blood draw \\
 \hline
 & It is possible to design specific assays to detect minimal quantities of tumor cells, for example the ones left behind after surgery. This is useful to detect minimal residual disease (MRD) and avoid tumor recurrence \\
 \hline
 & The collection of serial samples allows for example to track clonal evolution of the tumor over time, to catch treatment resistances early on and to monitor the patient's response to the treatment \\
 \hline
 & It can be used for early detection of cancer, many studies are trying to reach this objective \\
 \hline
\end{tabular}

\begin{tabular}{ | m{7cm}| m{7cm} | }
 \hline
 \multicolumn{2}{|c|}{Material availability} \\
 \hline
 From needle biopsies, biopsies, surgical resections (if some material is left after the clinical protocol and the patient agrees to a research protocol) & From circulating tumor cells, extracellular vesicles, cell-free DNA (the most interesting). In healthy donors there is ~4ng/ml cell-free DNA (below 10 anyway), in tumor patients ~100s ng/ml (but the range is really wide). The numbers are higher if the tumor is metastatic and the treatment is also very influential on the quantity of cfDNA. Tumor patients under treatment have cfDNA quantities comparable to healthy people. Anyway, cfDNA quantity is influenced by a number of factors in addition to cancers so it is not a good diagnostic feature by itself \\
 \hline
\end{tabular}

\begin{tabular}{ | m{7cm}| m{7cm} | }
 \hline
 \multicolumn{2}{|c|}{Tumor content} \\
 \hline
 Tumor content can be assessed with a microscope: the proportion of tumor cells compared to healthy cells is measured based on morphology with a simple staining of the tissue slide. So tumor content is assessed by counting cells and considering the magnification of the image. If subtyping is needed, a staining for markers is performed. Computational methods are also available & The fraction of circulating tumor DNA (ctDNA) is inferred with methods based on genomics (or possibly also methylation) \\
 \hline
\end{tabular}

\begin{tabular}{ | m{7cm}| m{7cm} | }
 \hline
 \multicolumn{2}{|c|}{Tumor ploidy/aneuploidy } \\
 \hline
 Inferred with cytogenetics, FISH, or from NGS data & Inferred based on genomics but it is quite tricky \\
 \hline

\end{tabular}

\section{Issues in the interpretation of cfDNA data}

\subsection{Normalization on tumor content}
When interpreting data from liquid biopsies, it is fundamental to contextualize a mutation after observing it. In order to associate a particular mutation to a particular diagnosis the signal has to be normalized based on tumor content. Without normalization, tumor content is the most influential variable on the patient's prognosis and this can be misleading. For example, one mutation could look like it is linked to a specific type of tumor when it is actually present in other types too but it is not detected due to the low tumor content of some samples \ref{fig:norm}. For this reason not all the literature available about liquid biopsies is reliable: lack of normalization leads to completely wrong conclusions. This applies to all kinds of assays: from microarrays to the sequencing of extracellular vesicles. \\

\begin{figure}[H]
    \includegraphics[width=0.4\linewidth]{norm.png}
    \caption{\label{fig:norm}The two samples have the same percentage of tumor cells, however the first one results negative for the marker because of the low tumor content}
\end{figure}

\subsection{Quantity of input material}
Another source of errors in the interpretation of cfDNA data is the amount of input material: if the patient's tumor content is high the results could be obtained with a limited amount of extracted nucleic acid, but if the tumor content is low, too little material can lower the chances of detecting tumor cells \ref{fig:quan}. The problem is that in most cases the tumor content is unknown before the analysis and this must be considered when designing an experiment. Usually the standard procedure is to begin with 2ml of plasma. If no tumor is detected, one should repeat the assay with more material (or sequence another vial and combine the results) to be sure that the tumor is not present and not just undetected. In some cases some information about the state of the patient is available: for example if a patient is in remission more material is required.

Keep in mind that if the sample is pure, 10 ng of DNA should correspond to around 1500 diploid tumor genomes.

\begin{figure}[H]
    \includegraphics[width=\linewidth]{quantity1.png}
    \caption{The same analyses are repeated with different quantities of starting material: for a patient with high tumor content the results do not change even with as little as 5ng of cfDNA}
\end{figure}
%
\begin{figure}[H]
    \includegraphics[width=\linewidth]{quantity2.png}
    \caption{Copy number signal correlates very well between different initial amounts of cfDNA for a sample with high tumor content}
\end{figure}



\section{SNV detection in liquid biopsies}

\textbf{Technical problems:}
\begin{itemize}
    \item PCR artifacts
    \item Sequencing errors: one mutation should be validated by multiple reads to be confirmed
    \item Problems related to the depth of coverage: the required coverage should be estimated considering the expected tumor content of the sample and deeper sequencing may be required
\end{itemize}
\textbf{Biological problems:}
\begin{itemize}
    \item Low tumor content: ctDNA/cfDNA ratio
    \item Clonal hematopoiesis (when a hematopoietic stem cell starts making cells with the same genetic mutation): to distinguish the signal coming from clonal hematopoiesis, compare it with what has been sequenced before from solid tumors. It is rare to observe something in liquid biopsy that has never been noticed in solid ones.
    \item Copy number variations and ploidy: with a whole genome duplication and a SNV only present on one allele, the signal corresponding to the mutation is only 25\% and has to be correctly interpreted.
    \item Intra-patient tumor heterogeneity: very low allelic fractions for SNVs that are not clonal can be difficult to observe
\end{itemize}

Multiple \textbf{tools} are available to detect SNVs. Each tool will probably give different results (or partially concordant ones). Each tool can be tuned to favour some types of calls, so the tuning parameters should be carefully selected.

\section{Requirements depend on the application}

\begin{tabular}{ | m{5cm}| m{9cm} | }
 \hline
 \textbf{Application} & \textbf{Requirements} \\
 \hline
 \begin{itemize}
     \item Early tumor detection
     \item MRD detection
     \item Recurrence detection
 \end{itemize} & Tumor quantity is low so a low signal is expected: higher quantity of starting material is required but there needs to be a balance between the number of false positives (with too much material) and false negatives (with too little) that can be produced \\
 \hline
 \begin{itemize}
     \item Tumor dynamics
     \item Treatment response
     \item Mechanisms of resistance
 \end{itemize} & The assay should be designed in order to be able to distinguish between different clones (sub clonality analysis) \\
 \hline
 Single biomarker assessment & The only important thing is to detect whether one point mutation is present or not, so in this case tumor content is not important. A targeted assay is used and specific locations associated with the SNV are sequenced as deep as possible to detect the mutation \\
 \hline
\end{tabular}

\section{Whole genome vs targeted sequencing}
Whole genome sequencing has higher computational cost, while targeted assays have higher sample preparation time. The sequencing cost is higher for whole genomes but it does not decrease evenly \ref{fig:wg}.

\begin{figure}[H]
    \includegraphics[width=0.7\linewidth]{wg.png}
    \caption{\label{fig:wg}Whole genome vs targeted sequencing}
\end{figure}


\section{Take-home message}
\textbf{Possible exam question}: what type of assay should be run and which are the requirements for a specific situation.\\

% #TODO MANCA LA PARTE DI QUANDO È ANDATA VIA A METÀ LEZIONE
