\graphicspath{{chapters/papers/06/images}}
\chapter{SNP panel identification assay (SPIA): a genetic-based assay for the identification of cell lines}

\section{Abstract}
Experiments reported in the scientific literature may contain pre-analytic errors due to inaccurate identities of the cell lines employed.
To address this a simple approach to enable accurate determination of cell line identify has been developed by genotyping SNP.

	\subsection{Introduction}
	Cell lines are important in the identification of therapeutic targets and in understanding molecular pathways related to drug-tumour interactions.
	One recognized risk in cell line maintenance is human error, either by mislabelling or cross-contamination.
	In an effort to identify latent cross-contamination or other errors SNPs are considered.
	SNPs as DNA markers have been shown to be well suited for different purposes such as animal identification, identification of population ancestry and for forensic purposes.
	The ability of high-density oligonucleotide arrays to genotype hundreds of thousands of SNP loci in parallel provides a molecular fingerprint of each sample.
	To this end an assay that employs 30 to 50 single loci and capable of distinguishing any two DNA sample has been developed.
	This assay can identify a given sample comparing its genotype with a reference dataset.


\section{Material and methods}

	\subsection{Genotype distance}
	To evaluate the similarity of two DNA sample the similarity measure $D$ is introduced.
	$D$ is proportional to the number of genotype mismatches between the samples and is normalized to the number of genotype calls available for both samples.
	Given a set of $N_{SNPs}$ of individual SNPs, $CL1$ and $CL2$ are ordered sets of genotype calls of two samples and $vN_{SNPs} - Card(T)$, where $T = \{i: cl1_i \neq NoCall\cap cl2_i\neq NoCall\}$.
	For $vN_{SNPs} > 0$, $D$ is defined as:

	$$D(CL1, CL2) = \frac{1}{vN_{SNPs}}\sum\limits_{i=1, \dots, N_{SNPs}}d(cl1_i, cl2_i)$$

	Where:

	$$d(cl1_i, cl2_i) = \begin{cases} 1 &if\ cl1_i\neq cl2_i\\ 0 &if\ cl1_i = cl2_i\lor cl_i = NoCall\end{cases}$$

	The distance is normalized over the number of available calls.
	Moreover the algorithm evaluates:

	\begin{multicols}{2}
		\begin{itemize}
			\item The count of mismatches where the two samples are homozygous for different alleles.
			\item The count of mismatches where one is homozygous and the other is heterozygous.
			\item The count of homozygous matches and the count of heterozygous matches.
		\end{itemize}
	\end{multicols}

	For each mismatch the algorithms reports the identifier of the sample with largest number of heterozygous calls.
	The implementation of the distance can be modified to weight different types of mismatch.

	\subsection{SNP panel selection procedure}
	Using a small number of SNPs samples can still be accurately distinguished.
	Initial filters on the selection of SNPs where:

	\begin{multicols}{2}
		\begin{itemize}
			\item SNPS with the rs identifier.
			\item SNPs not in intronic regions.
			\item SNPS represented on the 10K Affymetrix oligonucleotide array.
		\end{itemize}
	\end{multicols}

	On the training set the minor allele frequency, the heterozygosity rate and the call rate for each SNP across all sample have been computed.
	Then SNP satisfying the Hardy-Weinberg equilibrium applying elastic boundaries and having SNP call rates more than $80\%$ have been filtered.
	Then on the test set, the heterozygosity rate of the identified SNPs has been computed.
	At each iteration a variable number of SNPs has been computed.
	SNPs have been ranked according to the selection rate.

	\subsection{SPIA probabilistic test on cell line genotype distance}
	A double probabilistic test to apply on the genotype distance is applied to discern when two cell lines are close enough to be called similar.
	The test score depends on the number of matches and on the total number of SNPs evaluated.
	The test relies on the probability of the evaluated distance belonging to the population of real matched pairs or to the population of real non-pairs.
	If the output is not clear a second panel of SNPs would need to be investigated.
	If the SNPs are independent and the genotype call probability being the same at each SNP, the probability of having $k$ matches out of $N$ SNPs follows the binomial distribution:

	$$P_k = \binom{N}{k} P^kQ^{N-k} = \frac{N!}{k!(N-k)!}P^kQ^{N=k}$$

	Where $P$ and $Q$ are the probability of match and mismatch and $N$ is $vN_{SNPs}$.
	By knowing the probability of match at a single SNP for a real matched pair $P_M$ and for a non-matched pair $P_{non-M}$ the distribution of real matched pair and non-pair can be drawn.
	For a given $vN_{SNPs}$ then areas corresponding to ``different'', ``uncertain'' and ``similar'' can be defined.
	The area limits depend on the level of confidence needed.
	The mean number of successes $k_{mean}$ is equal to $NP_M$ and the standard deviation $sd_{kmean} = \sqrt{NP_M(1-P_M)}$.
	The probability that a distance measurement falls within $M$ standard deviations from the mean is given by the integral of the distribution function.
	By setting $m$ the area limits can be defined.
	The smaller the number of SNPs the narrower the region of uncertainty and the higher the probability of making an incorrect call.
